runtime/doc/options.txt

```diff
@@ -1217,6 +1217,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 	without saving.  For writing there must be matching |BufWriteCmd|,
 	|FileWriteCmd| or |FileAppendCmd| autocommands.
 
+						*'busy'* *'nobusy'*
+'busy'			boolean	(default off)
```

Comment:
A simple boolean is probably the right interface. Anything else may be overengineering. Because the use-case is for plugins that "own" an application-specific buffer.